.gitignore

```diff
@@ -14,3 +14,9 @@ out/
 /CMakeSettings.json
 CMakeCache.txt
 cmake.check_cache
+
+#VSCode Debugger files
+Intern/rayx/src/results.json
+Intern/rayx-core/src/Tracer/results.json
+Intern/rayx-core/src/VulkanEngine/results.json
+Intern/rayx-core/src/VulkanEngine/Init/results.json
```

Comment:
I think this could be `**/results.json` to be more general, if I recall correctly. It's a good idea to put this in .gitignore btw!